docs/contributing.rst

```diff
@@ -35,7 +35,7 @@ Now, in your virtual environment, ``twine`` is pointing at your local copy, so
 when you make changes, you can easily see their effect.
 
 We use `tox`_ to run tests, check code style, and build the documentation.
-To install ``tox`` in your active your virtual environment, run:
+To install ``tox`` in your active virtual environment, run:
```

Comment:
Following up on https://github.com/pypa/twine/pull/633/files#r429546104 and https://github.com/pypa/twine/pull/633/files#r429551908:

Except for the typo, I'm inclined to leave this instruction as-is for now, since we resolved the issue @deveshks was having. This section seems sufficient a new contributor who only has Python 3 installed to set up a complete development environment for Twine.